_posts/2020-09-15-open-sourcing-a-private-project.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "How to Open Source a Closed Source Project"
+date: 2020-09-29 11:00:00
+categories: ["open-source"]
+author: "fionaDL"
```

Comment:
This should be:

```suggestion
author: "fionadl"
```

Reply:
@FionaDL btw, if you prefer to display `fionaDL`, you can update it in the config and in the older posts

Reply:
@lubc It doesn't matter to me :)